API/API/settings.py

```diff
@@ -131,3 +132,8 @@
 # https://docs.djangoproject.com/en/5.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+SIMPLE_JWT = {
+    "ALGORITHM" : "RSA512", 
+    "SIGNING_KEY" : os.getenv('JWT_SECRET', 'defaultkey'), 
```

Comment:
maybe add a #TODO to remove this default value as it may be insecure when user won't provide `JWT_SECRET`

Reply:
added a todo and the default key is now a valid rsa512, example key is 2048 bit if that makes any difference.